app/jobs/create_onboarding_notifications_job.rb

```diff
@@ -0,0 +1,53 @@
+class CreateOnboardingNotificationsJob < ApplicationJob
+  queue_as :background
+
+  OnboardingEmail = Struct.new(:day, :slug, keyword_init: true) do
+    def notification_type
+      "onboarding_#{slug}".to_sym
+    end
+  end
+
+  # 0 would be immediately after signing up
+  # 1 is a day after signing up, etc
+  #
+  # Each slug should be paired with a User::Notifications::Onboarding{$SLUG}Notification
+  EMAILS = {
+    1 => :community,
+    3 => :fundraising
+  }.map { |day, slug| OnboardingEmail.new(day:, slug:) }.freeze
+
+  # For each email we get all the users that signed up between
+  # n days ago and n+safety_offset days ago.
+  # So for example, for a day 3 email, if SAFETY_OFFSET is 1,
+  # we check anyone that signed up between days 3 and 4.
+  #
+  # This gives us a security blanket that if our scripts don't
+  # run for a period of time, no-one gets missed. But after the safety
+  # period we don't end up spamming old users. All onboarding notifications
+  # only send once, so this is safe to run multiple times.
+  SAFETY_OFFSET = 1
```

Comment:
Maybe make it explicit that this is about days?

```suggestion
    SAFETY_OFFSET_IN_DAYS = 1
```